src/stores/pacbioPoolCreate.js

```diff
@@ -251,15 +261,17 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
     requestList: (state) => (ids) => {
       const requests = state.resources.requests
       const selectedRequests = state.used_aliquots
+      let val = []
       if (ids) {
-        return ids.map((id) => {
+        val = ids.map((id) => {
           return { ...requests[id], selected: !!selectedRequests[`_${id}`] }
```

Comment:
did you figure out why underscores are used for ids? I assume its because they are stored in the same place as requests in the store and need some way of differentiating them.

Reply:
Yes, I believe it is used to differentiate between the ones available and ones selected, I believe,  as all of them are stored in same place. The selected ones are having the _ prefix.